contrib/container-fedora-25/custodia.conf

```diff
@@ -0,0 +1,42 @@
+[DEFAULT]
```

Comment:
It's config file for https://pypi.python.org/pypi/custodia.ipa, from that page. Until `custodia.ipa` is merged and made part of custodia, we cannot really update the default config file.

Overall it'd be much better to just support `custodia.conf.d/`and let consumers drop in snippets that they need, rather than fiddle with the whole config files.